source/core.py

```diff
@@ -230,6 +231,63 @@ def getWxLangOrNone() -> Optional['wx.LanguageInfo']:
 	return wxLang
 
 
+def triggerNVDAExit():
+	preNVDAExit.notify()
+	# ensure NVDA only runs exit procedures once
+	handlers = list(preNVDAExit.handlers)  # don't mutate .handlers directly with unregister while iterating
+	for handler in handlers:
+		preNVDAExit.unregister(handler)
+
+
+def closeAllWindows():

```

Comment:
It seems like this should never be called directly, only via `preNVDAExit` Action. If so, please add a note to the doc string.

I also think this function name should be prefixed with an underscore, as a convention to signify their "protected" state. Typically anything that we don't want add-ons to use should be prefixed with an underscore (or in a class that is named that way).